### PR TITLE
add jax support for Lotka Volterra example

### PR DESCRIPTION
**Pull request by HGangloff**

### CODE REVIEW

File: examples/pinn_forward/Lotka_Volterra.py [hunk 1]
```diff
@@ -1,128 +1,159 @@
...
+# def input_transform(t):
+#     return paddle.concat(
+#         (
+#             paddle.sin(t),
+#         ),
+#         axis=1,
+#     )
+# Backend jax
+# def input_transform(t):
+#     if t.ndim == 1:
```
Comment (lululxvi) [inline examples/pinn_forward/Lotka_Volterra.py:94]: Why do we need `if t.ndim == 1:`?

File: examples/pinn_forward/Lotka_Volterra.py [hunk 2]
```diff
@@ -1,128 +1,159 @@
...
+#     y2 = y[:, 1:2]
+#     return torch.cat([y1 * torch.tanh(t) + 100 / ub, y2 * torch.tanh(t) + 15 / ub], dim=1)
+# Backend paddle
+# def output_transform(t, y):
+#     y1 = y[:, 0:1]
+#     y2 = y[:, 1:2]
+#     return paddle.concat([y1 * paddle.tanh(t) + 100 / ub, y2 * paddle.tanh(t) + 15 / ub], axis=1)
+# Backend jax
+# def output_transform(t, y):
+#     if y.ndim == 1:
```
Comment (lululxvi) [inline examples/pinn_forward/Lotka_Volterra.py:122]: Why `if y.ndim == 1:`?

**Comment by HGangloff** [comment]
We need those tests because before vmapping the input for jax function has no batch dimension. Note that in `output_transform`  the condition is not necessary but the `.squeeze()` remains.

**Comment by lululxvi** [comment]
In the code, `t.ndim == 1` should be either always `True`, or always `False`. You can keep `t = t[None]` if needed, but `if t.ndim == 1:` should be removed.

**Comment by HGangloff** [comment]
No it varies, I admit that I still don't get why... something in the Jax backend complicates the use of input_transform and output_transform

**Pull request merged**